db/seeds/muscle_descriptions.rb

```diff
@@ -0,0 +1,77 @@
+
+  Strong trapezius muscles are commonly seen on most athletes and are not usually considered a muscle to be focused on related to other groups for the average individual. However, it should be noted that trapezius perform many everyday functions and are a great benefit for everyone. Movements that involved moving your shoulder blades up or together as well as shrugging functions are all major functions."
+
+  CALVES =
+  "The calf muscle is divided into two general parts. One main part is the gastrocnemius, which is U-shaped. The function of this part is to raise your body in an upright position up by the toes. The other part is the soleus, which is below the gastrocnemius. It is used as well to raise your body, but in a position when your knees are bent.
+
+  The calves are a very stubborn muscle to train. It is a fast twitch muscle which means that you have to train it with more intensity. You must fully stretch the calves at the bottom portion of the repetition and contract them in the full range of motion. It is recommended that you hold the contraction for a few seconds to really stimulate all of the calf muscle tissues. There will be a build up of lactic acid as you perform the repetition, however this is the point where you must do a few more reps to get results. Remember, train your calves like they are any other body part, do not put them aside."
+
+  HAMSTRINGS =
+  "The hamstring muscle is located directly at the back of your thigh and works in unison with your quadriceps as well as your glutes and calves. This promotes proper movement of the entire thigh muscles. It is actually composed of three posterior thigh muscles of the semitendinosus, semimembranosus and the biceps femoris.
+
+  The hamstrings muscles play a large role in flexibility of the lower body. Most effective exercises generally are curls at the knee as well a dead-lift type movements. This large muscle group should not be neglected as it will improve mobility and long term flexibility."
+
+  QUADS =
+  "The quadriceps femoris muscle is composed of four prevailing muscles located at the front of the thigh. These four muscle regions are the Rectus femoris, Vastus lateralis, Vastus medialis and finally the Vastus intermedius. The strongest muscle in the body, the quadriceps is responsible for the extension of the knee.
+
+  There are several leg exercise that successfully target the quads. The most effective of the bunch tend to be the leg press as well as the squat. One excellent isolation exercise is the leg extension."
+
+  GLUTES =
+  "Gluteus maximus, also commonly referred to as the glutes, this large and strong muscle group located on your buttocks is the most superficial of the group. It is responsible for many movements originating from the trunk as well as thigh movements.
```

Comment:
Line is too long. [250/80]<br>Indent the first line of the right-hand-side of a multi-line assignment.<br>Freeze mutable objects assigned to constants.